pkg/kubectl/cmd/util/cached_discovery.go

```diff
@@ -25,8 +25,8 @@ import (
 	"time"
 
 	"github.com/emicklei/go-restful-swagger12"
-	"github.com/go-openapi/spec"
 	"github.com/golang/glog"
+	openapi_v2 "github.com/googleapis/gnostic/OpenAPIv2"
```

Comment:
This is mostly because my `goimports` was broken. I've fixed it and I'm using the default name now (which happens to also openapi_v2 ;-)